lib/liquid/usages/try_variables.rb

```diff
@@ -0,0 +1,26 @@
+module Liquid
+  class Context
+    alias try_variable_find_in_environments_original try_variable_find_in_environments
+
+    def try_variable_find_in_environments_usage(key, raise_on_not_found:)
+      Usage.track("Using try_variable_find_in_environment")
+      @environments.each do |environment|
+        found_variable = lookup_and_evaluate(environment, key, raise_on_not_found: raise_on_not_found)
+        if !found_variable.nil? || @strict_variables && raise_on_not_found
+          return found_variable
```

Comment:
Oh, I guess these early returns make it hard to have a simple wrapper with usage tracking that doesn't duplicate functionality.  That means that if we change the wrapped method, then we would need to make sure to also change this method, which could be easy to miss if we don't test with usage tracking enabled.

Should we instead add the usage tracking calls directly in the code itself?  That way we would avoid code duplication and simplify maintenance of that code.  E.g. the call to `Usage.track` could just be a no-op if usage tracking isn't enabled.

Reply:
I guess my reasoning for trying to keep this out of the code was to avoid people just leaving it there. However this likely has enough eyes on it for that not to matter.